shared/chat/inbox/row/small-team/index.js

```diff
@@ -134,30 +149,28 @@ class SmallTeam extends React.PureComponent<Props, State> {
 }
 
 const styles = styleSheetCreate({
-  container: {flexShrink: 0, height: RowSizes.smallRowHeight},
+  container: {flexShrink: 0, height: InboxSizes.smallRowHeight},
   conversationRow: {
     ...globalStyles.flexBoxColumn,
     flexGrow: 1,
     height: '100%',
     justifyContent: 'center',
     paddingLeft: 8,
     paddingRight: 8,
+    width: InboxSizes.inboxWidth - 48 - 16,
```

Comment:
Where's the - 48 - 16 coming from? A comment would be helpful here

Reply:
Hmm, this doesn't seem to be doing anything actually.. I'll get rid of it